apps/pay/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Install dependencies only when needed
+FROM node:20-alpine AS deps
```

Comment:
This file can probably be redone similar to ones in `app/dashboard` and `apps/consent`